dsc_lib/src/util.rs

```diff
@@ -3,6 +3,26 @@
 
 use crate::dscerror::DscError;
 use serde_json::Value;
+use std::fs::File;
+use std::io::BufReader;
+use std::path::PathBuf;
+use std::path::Path;
+use std::env;
+use tracing::debug;
+
+pub struct DscSettingValue {
+    pub setting:  serde_json::Value,
```

Comment:
Should be able to do this and make it more clean unless `Value` conflicts with a different type

```suggestion
    pub setting:  Value,
```

Reply:
Updated, even though I don't like it. :)